backend/app/services/execution/inprocess_executor.py

```
@@ -0,0 +1,477 @@
+            # Note: The agent's emitter will have already sent DONE/ERROR events
+            # through the bridge transport, so we don't need to emit them again here
+
+            # Cleanup: Close Claude Code client after task completion
+            # This releases resources since standalone mode doesn't need to keep
+            # clients alive for container reuse like Docker mode does
+            await self._cleanup_task_clients(request.task_id)
+
+        except Exception as e:
+            logger.exception(
+                f"[InprocessExecutor] Execution error: "
+                f"task_id={request.task_id}, error={e}"
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)